### ISSUE TITLE
ingress-gce-glbc does not work with Kubernetes 1.22

### ISSUE THREAD

**Issue by jprzychodzen** [kubernetes/ingress-gce#1414]
Even with image `gcr.io/k8s-ingress-image-push/ingress-gce-glbc-amd64:v1.11.0` I still see CrashLoop of l7-lb-controller when I run kubernetes/kubernetes#99840. There is following ​error message: 

```
F0422 15:46:51.812286       1 main.go:174] app.NewNamer(ctx.KubeClient, "", "") = the server could not find the requested resource (get ingresses.extensions)
goroutine 1 [running]:
k8s.io/klog.stacks(0xc000621200, 0xc0001061c0, 0x9e, 0xd4)
       ​/go/pkg/mod/k8s.io/klog@v1.0.0/klog.go:875 +0xb8
k8s.io/klog.(*loggingT).output(0x42006e0, 0xc000000003, 0xc0000ee000, 0x4146a0f, 0x7, 0xae, 0x0)
       ​/go/pkg/mod/k8s.io/klog@v1.0.0/klog.go:826 +0x330
k8s.io/klog.(*loggingT).printf(0x42006e0, 0x3, 0x2beff31, 0x29, 0xc000507a50, 0x3, 0x3)
       ​/go/pkg/mod/k8s.io/klog@v1.0.0/klog.go:707 +0x14b
k8s.io/klog.Fatalf(...)
       ​/go/pkg/mod/k8s.io/klog@v1.0.0/klog.go:1276
main.main()
       ​/go/src/k8s.io/ingress-gce/cmd/glbc/main.go:174 +0x174b
```

This is a blocker for kubernetes/kubernetes#99840 - because of this error, test pull-kubernetes-e2e-gce-100-performance is failing.

/assign @swetharepakula
/cc @wojtek-t
/cc @deads2k

**Comment by swetharepakula** [comment]
The newest image that was updated only had the CRD API change.

Work is actively being done to convert the controller to use the `networking/v1` API which is the cause of the error above.

**Comment by swetharepakula** [comment]
Tag v1.12.0 has been cut. This version removes Ingress beta APIs.
https://github.com/kubernetes/kubernetes/pull/101772 updates the image used in kubernetes/kubernetes

**Comment by swetharepakula** [comment]
fixed by #1416 

/close
